core/layout_api.php

```diff
@@ -334,6 +334,7 @@ function layout_body_javascript() {
 
 	# ace theme scripts
 	html_javascript_link( 'ace.min.js' );
+	html_javascript_link( 'clipboard.impl.js' );
```

Comment:
This is not an ace theme script... Add a comment.

I would suggest to load this script only on changelog page. Currently it's loaded on every single page but it's not actually used anywhere else.

Finally, I would suggest renaming the script, e.g. `changelog.js`